Random failure occurs in integration test "TestBundleErrorWhenOVSRestart"

**Describe the bug**
The integration tests failed quite often recently because of timeout (10min). And the it is stuck at case "TestBundleErrorWhenOVSRestart"

**To Reproduce**
Run integration test.

**Expected**
All tests should pass

**Actual behavior**
The integration tests failed, and the failed case is "TestBundleErrorWhenOVSRestart". After dumping goroutines, one goroutine info shows like following,
```
sync.runtime_Semacquire(0xc000044e08)
	/usr/local/go/src/runtime/sema.go:56 +0x42
sync.(*WaitGroup).Wait(0xc000044e00)
	/usr/local/go/src/sync/waitgroup.go:130 +0x64
github.com/vmware-tanzu/antrea/test/integration/ovs.TestBundleErrorWhenOVSRestart(0xc00014c300)
	/usr/src/github.com/vmware-tanzu/antrea/test/integration/ovs/ofctrl_test.go:461 +0x624
testing.tRunner(0xc00014c300, 0x16493f0)
	/usr/local/go/src/testing/testing.go:909 +0xc9
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:960 +0x350
```


**Versions:**
Please provide the following information:
 - Antrea version (Docker image tag): 0.9.0+
 - Kubernetes version (use `kubectl version`). If your Kubernetes components have [different versions]

There should be some bug when resolving the goroutine leak issue in ofnet, which might cause the integration test failed and report a new error like,
```
panic: send on closed channel

goroutine 30736 [running]:
github.com/contiv/ofnet/ofctrl.(*OFSwitch).publishMessage.func1(0xc001f669c0, 0x1, 0x2cb00000000, 0x1)
	/tmp/gopath/pkg/mod/github.com/wenyingd/ofnet@v0.0.0-20200911061943-60f892dbc243/ofctrl/ofSwitch.go:463 +0x49
created by github.com/contiv/ofnet/ofctrl.(*OFSwitch).publishMessage
	/tmp/gopath/pkg/mod/github.com/wenyingd/ofnet@v0.0.0-20200911061943-60f892dbc243/ofctrl/ofSwitch.go:462 +0xfe
FAIL	github.com/vmware-tanzu/antrea/test/integration/ovs	6.924s
FAIL
```

--- a/go.mod
+++ b/go.mod
@@ -61,7 +61,7 @@ require (
 replace (
 	// antrea/plugins/octant/go.mod also has this replacement since replace statement in dependencies
 	// were ignored. We need to change antrea/plugins/octant/go.mod if there is any change here.
-	github.com/contiv/ofnet => github.com/wenyingd/ofnet v0.0.0-20200911061943-60f892dbc243
+	github.com/contiv/ofnet => github.com/wenyingd/ofnet v0.0.0-20200911061943-57045ae085da
 	// fake.NewSimpleClientset is quite slow when it's initialized with massive objects due to
 	// https://github.com/kubernetes/kubernetes/issues/89574. It takes more than tens of minutes to
 	// init a fake client with 200k objects, which makes it hard to run the NetworkPolicy scale test.

